2022-06-27 16:12:17,953 INFO    MainThread:85128 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_setup.py:_flush():76] Loading settings from /home/compu/jh/project/colon_compare/scripts/logs/evaluations/runs/default/2022-06-27_16-11-54/wandb/settings
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-06-27 16:12:17,954 WARNING MainThread:85128 [wandb_setup.py:_flush():76] Could not find program at ../test.py
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../test.py'}
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_init.py:_log_setup():428] Logging user logs to ./wandb/run-20220627_161217-1k0vxdjg/logs/debug.log
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_init.py:_log_setup():429] Logging internal logs to ./wandb/run-20220627_161217-1k0vxdjg/logs/debug-internal.log
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_init.py:init():462] calling init triggers
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {}
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_init.py:init():515] starting backend
2022-06-27 16:12:17,954 INFO    MainThread:85128 [wandb_init.py:init():519] setting up manager
2022-06-27 16:12:17,956 INFO    MainThread:85128 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-27 16:12:17,957 INFO    MainThread:85128 [wandb_init.py:init():525] backend started and connected
2022-06-27 16:12:17,961 INFO    MainThread:85128 [wandb_init.py:init():596] updated telemetry
2022-06-27 16:12:17,961 INFO    MainThread:85128 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-06-27 16:12:18,412 INFO    MainThread:85128 [wandb_run.py:_on_init():1924] communicating current version
2022-06-27 16:12:18,472 INFO    MainThread:85128 [wandb_run.py:_on_init():1928] got version response upgrade_message: "wandb version 0.12.19 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-06-27 16:12:18,472 INFO    MainThread:85128 [wandb_init.py:init():662] starting run threads in backend
2022-06-27 16:12:18,591 INFO    MainThread:85128 [wandb_run.py:_console_start():1898] atexit reg
2022-06-27 16:12:18,591 INFO    MainThread:85128 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-06-27 16:12:18,592 INFO    MainThread:85128 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-06-27 16:12:18,593 INFO    MainThread:85128 [wandb_run.py:_redirect():1832] Redirects installed.
2022-06-27 16:12:18,594 INFO    MainThread:85128 [wandb_init.py:init():688] run started, returning control to user process
2022-06-27 16:12:18,720 INFO    MainThread:85128 [wandb_run.py:_config_callback():1136] config_cb None None {'ckpt_path': '/home/compu/jh/project/colon_compare/scripts/logs/experiments/runs/default/2022-06-27_09-45-08/checkpoints/epoch_025.ckpt'}
2022-06-27 16:13:22,450 WARNING MsgRouterThr:85128 [router.py:message_loop():76] message_loop has been closed
